pages/model/[uuid].tsx

```diff
@@ -81,6 +90,8 @@ function Model() {
   const { versions, isVersionsLoading, isVersionsError } = useGetModelVersions(uuid)
   const { version, isVersionLoading, isVersionError, mutateVersion } = useGetModelVersion(uuid, selectedVersion)
   const { deployments, isDeploymentsLoading, isDeploymentsError } = useGetModelDeployments(uuid)
+  const [deleteConfirmOpen, setDeleteConfirmOpen] = useState<boolean>(false)
+  const [deleteConfirmContent, setDeleteConfirmContent] = useState<JSX.Element>(<Box />)
```

Comment:
Instead of defaulting to an empty `Box` you should use `null`. The difference is that a `Box` will be rendered on the page as an empty `div` which, however minor, is work that the browser has to do. We should always aim to minimise the work that the browser has to do as that is a limited resource. `null` on the other hand will simply not be rendered at all.